assemblyline/templates/ui.yaml

```diff
@@ -187,8 +187,10 @@ spec:
           {{ end}}
           volumeMounts:
           {{ include "assemblyline.coreMounts" . | indent 12 }}
+            - name: ui-tls-cert
```

Comment:
We'd have to wrap the volumeMount around the same condition as the volume I think